apps/client/src/app/model/garland-tools/item.ts

```diff
@@ -12,6 +12,7 @@ export class Item implements I18nData {
   en: I18nDataRow;
   de: I18nDataRow;
   ja: I18nDataRow;
+  zh: I18nDataRow;
```

Comment:
garland tools doesn't provide zh item names, same for ko, which is why ko isn't in this interface ^^

Reply:
I must forget this. Thanks and I have fixed this.